index.html

```diff
@@ -57,17 +57,17 @@
 
 <ul>
 <li><a href="https://chromium.googlesource.com/experimental/chromium/blink/+/refs/wip/bajones/webvr">Blink</a> and <a href="https://chromium.googlesource.com/experimental/chromium/src/+/refs/wip/bajones/webvr">Chromium</a> (Both part of the Chrome codebase)</li>
```

Comment:
@toji I'm pretty ignorant when it comes to Chromium knowledge, so correct me if I'm wrong. iirc, the blink repo was merged into the chromium one?